unity-renderer/Assets/DCLPlugins/EquippedEmotes/EquippedEmotesPlugin.cs

```diff
@@ -0,0 +1,114 @@
+using DCL.Helpers;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.EquippedEmotes
+{
+    /// <summary>
+    /// Plugin feature that initialize the Equipped Emotes feature.
+    /// </summary>
+    public class EquippedEmotesPlugin : IPlugin
+    {
+        internal const string PLAYER_PREFS_EQUIPPED_EMOTES_KEY = "EquippedNFTEmotes";
+
+        internal DataStore_EmotesCustomization emotesCustomizationDataStore => DataStore.i.emotesCustomization;
+
+        public EquippedEmotesPlugin()
+        {
+            LoadDefaultEquippedEmotes();
+
+            emotesCustomizationDataStore.isInitialized.OnChange += OnEmotesCustomizationInitialized;
+            OnEmotesCustomizationInitialized(emotesCustomizationDataStore.isInitialized.Get(), null);
+
+            emotesCustomizationDataStore.avatarHasBeenSaved.OnChange += SaveEquippedEmotesInLocalStorage;
```

Comment:
why do we need this?

Cannot we just use `emotesCustomizationDataStore.equippedEmotes`?